app/src/main/java/com/duckduckgo/app/settings/SettingsActivity.kt

```diff
@@ -294,6 +294,7 @@ class SettingsActivity : DuckDuckGoActivity() {
     private fun observeViewModel() {
         viewModel.viewState()
             .flowWithLifecycle(lifecycle, Lifecycle.State.RESUMED)
+            .distinctUntilChanged()
```

Comment:
@aitorvs  do we need this? The flow already doesn't emit duplicates, right?

Reply:
We don't strictly need it, no strong opinions, I can remove it.
The only benefit is that without looking at the VM implementation it's clear we don't trigger any logic upon same emissions (which may not always be the case if we change something in the VM)

Lmk if you want me to remove it

Reply:
Let's keep it.